Changes.md

```diff
@@ -2,6 +2,94 @@
 
 ## H2O
 
+### Zermelo (3.32.0.1) - 10/xx/2020
+
+Downlaod at: <a href='http://h2o-release.s3.amazonaws.com/h2o/rel-zermelo/1/index.html'>http://h2o-release.s3.amazonaws.com/h2o/rel-zermelo/1/index.html</a>
+
+#### Bug
+<ul>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-5932'>PUBDEV-5932</a>] -         Added support for Python 3.7+.
+</li>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7667'>PUBDEV-7667</a>] -         Fixed StackedEnsemble’s retrieval of the seed parameter value.
+</li>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7746'>PUBDEV-7746</a>] -         Deserialization values of MOJO ModelParameter now work when the Value Type is int().
+</li>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7760'>PUBDEV-7760</a>] -         H2O no longer uses lazy-loading for sequential zip parse.
+</li>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7762'>PUBDEV-7762</a>] -         Updated model_type argument names for Rulefit in R.
+</li>
+</ul>
+    
+#### New Feature
+<ul>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7241'>PUBDEV-7241</a>] -         Quantile and Tweedie distributions added to monotone constraints.
```

Comment:
So can be Tweedie mention here or not if this feature was added in some previous fix release? @michalkurka

Reply:
yeah, it was already mentioned previously - we don't need to mention again